start.sh

```diff
@@ -19,98 +19,101 @@ printHelp() {
 	echo ""
 	echo "Options:"
 	echo "    -h, --help        print this help / usage"
-	echo "    -a, --agent       specify a local agent package"
-	echo "    -ct, --controller specify a local controller package"
-	echo "    -cn, --connector  specify a local connector package"
-	echo "    --no-cache        prevent the usage of cache during the build step"
+	echo "    -a, --agent       specify a local agent image"
+	echo "    -ct, --controller specify a local controller image"
+	echo "    -cn, --connector  specify a local connector image"
     echo ""
     echo "Arguments:"
 	echo "    [environment]     setup demo application, optional, default: iofog"
 	echo "                      supported values: iofog, tutorial"
 }
 
-checkForComposeFile() {
-    local ENVIRONMENT="$1"
-    local COMPOSE_FILE="docker-compose-${ENVIRONMENT}.yml"
-    if [[ ! -f "${COMPOSE_FILE}" ]]; then
-        echoError "Environment configuration for \"${ENVIRONMENT}\" does not exist!"
-        exit 2
+startIofog() {
+    # If stack is running, skip
+    local CONTROLLER_CONTAINER_ID=$(docker ps -q --filter="name=iofog-controller")
+    if ! [[ -z $CONTROLLER_CONTAINER_ID  ]]; then
+        return
     fi
+
+    echo "---
+apiVersion: iofog.org/v1
+kind: ControlPlane
+metadata:
+  name: local-ecn
+spec:
+  images:	
+    controller: $CONTROLLER_IMAGE
+  iofogUser:
+    name: test
+    surname: local
+    email: user@domain.com
+    password: '#Bugs4Fun'
+  controllers:	
+  - name: local-controller
+    host: localhost
+---
+apiVersion: iofog.org/v1
+kind: Connector
+metadata:
+  name: local-connector
+spec:
+  host: localhost
+  image: $CONNECTOR_IMAGE
+---
+apiVersion: iofog.org/v1
+kind: Agent
+metadata:
+  name: local-agent
+spec:
+  host: localhost
+  image: $AGENT_IMAGE
+" >| init/iofog/local-stack.yaml
```

Comment:
Would it not be less error prone to overwrite? In case someone has an old version and then runs the new one